src/main/java/pl/nowekolory/objectifyxlsx/cell/CellStyleCreator.java

```diff
@@ -40,18 +35,17 @@ public static CellStyle createFormatedCellStyle(Workbook workbook, Boolean round
 
     public static CellStyle createDateTimeCellStyle(Workbook workbook, String format){
         var dateTimeCellStyle = workbook.createCellStyle();
-        var stringFormat = getFormat(format, defaultDateTimeFormat);
-        var dateTimeFormat = workbook.getCreationHelper().createDataFormat().getFormat(stringFormat);
-        dateTimeCellStyle.setDataFormat(dateTimeFormat);
+        var createHelper = workbook.getCreationHelper();
+        dateTimeCellStyle.setDataFormat(
+                createHelper.createDataFormat().getFormat(defaultDateTimeFormat));
         return dateTimeCellStyle;
     }
 
     public static CellStyle createDateCellStyle(Workbook workbook, String format){
```

Comment:
Z tego co widzę to nie używasz już tutaj parametru format. Podobnie metoda wyżej

Reply:
Ogólnie może zostaw stare metody a to co teraz dorób jako osobne, dla wstecznej kompatybilności